assets/js/components/SettingsNotice/SettingsNoticeMultiRow.js

```diff
@@ -17,54 +17,43 @@
  */
 
 /**
- * External dependencies
+ * WordPress dependencies
  */
-import PropTypes from 'prop-types';
+import { Fragment } from '@wordpress/element';
 
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { TYPE_WARNING, TYPE_INFO, TYPE_SUGGESTION, getIconFromType } from './utils';
+import PropTypes from 'prop-types';
```

Comment:
@aaemnnosttv external dependencies should be defined higher than wordpress deps.

Reply:
Oops 🤦